cmake/SPATH_ADD_TEST.cmake

```diff
@@ -0,0 +1,18 @@
+function(SPATH_ADD_TEST name args outputs)
+
+  # job launcher
+  if(NOT(${VELOC_RESOURCE_MANAGER}))
+#    set(test_param mpirun -np 1)
+    set(test_param srun -N 3 -n 3)
+  elseif(${VELOC_RESOURCE_MANAGER} STREQUAL "NONE")
+    set(test_param mpirun -np 1)

```

Comment:
Do we want to test with multiple procs for mpirun and jsrun?

Reply:
for jsrun, the idea is that the "-nnodes 3" parameter set in VELOC_RESOURCE_MANAGER, in combination with this setting, will start a process on each of the 3 nodes. I am not sure how it would work for mpirun. Should I just saet it to "test_param mpirun -np 3"?

Reply:
Ah, ok.